src/views/fragments/AboutContent/styled.components.js

```diff
@@ -0,0 +1,46 @@
+import styled from "styled-components";
+
+export const HomeContentWrapper = styled.div`
```

Comment:
be mindful to change the name when copying over, but gonna refactor all of this into a unified styled soon